### ISSUE TITLE
Performance verbesserungen

### ISSUE THREAD

**Issue by tabacha** [ADFC-Hamburg/adfc-t30-api#90]
Moin,
der Call für "Gib mir alle Institutions" dauert recht lange (ist ja auch klar, sind viele Daten).

Ich würde mir wünschen, dass die API mir bei diesem Call im Ergebnis eine ID oder einen Timestamp (möglichst als Integer, dann muss man nicht Parsen) zurück liefert, wann eine Institution das letzte mal geändert wurde.

Das geht notfalls auch als extra Call, der sollte dann aber nur den Timestamp und nicht eine Änderungshistorie zurückliefern.

Außerdem sollt es einen Call geben: Gib mir alle Instituionen aber nur, wenn sie sich gegenüber einem Timestamp  (Paramter) geändetert haben. Alternativ wäre das auch mit dem oben genanten extra Call möglich.

Dann könnten wir im Frontend einen Institution Cache speichern und nur nur aktualisieren, wenn es Aktualisierungen gibt.

**Comment by bensteffen** [comment]
Jo, alle klar. Ich setzt mich mal dran die Tage. Sollte easy sein, da wir ja schon alle Änderungen der Institutionen mit dem EntityMonitor tracken.

**Comment by tabacha** [comment]
Sorry, das hat sich überschnitten. Ich habe auch schon eine Änderung gemacht, siehe Pull-Request.

**Comment by bensteffen** [comment]
Macht nix, hab ja auch nix von mir hören lassen...
